translator/translate/otel/exporter/awscloudwatch/testdata/config.json

```diff
@@ -1,6 +1,10 @@
 {
   "metrics": {
     "metrics_collected": {
+      "collectd": {
+        "collectd_security_level": "none",
```

Comment:
what is this parameter for ?

Reply:
I added this in my manual testing to simplify so that no auth file was needed. Since I tested with that, I used it here too.

Reply:
since we are not actually starting up a collectd port listening server up, I dont think this will be needed. This unit tests only checks the agent translator

Reply:
Done.